src/commands/base-command.ts

```diff
@@ -26,6 +26,7 @@ import {
   exit,
   getToken,
   log,
+  netlifyCliVersion,
```

Comment:
nit: This name feels a bit redundant and overly verbose to me, since everything here pertains to the Netlify CLI and there's no need to make that explicit with a prefix. Could we use something like `version`?